cr.sh

```diff
@@ -199,12 +199,8 @@ lookup_changed_charts() {
     local changed_files
     changed_files=$(git diff --find-renames --name-only "$commit" -- "$charts_dir")
 
-    local fields
-    if [[ "$charts_dir" == '.' ]]; then
-        fields='1'
-    else
-        fields='1,2'
-    fi
+    local depth=$(( $(tr "/" "\n" <<< "$charts_dir" | wc -l) + 1))
```

Comment:
```suggestion
    local depth=$(( $(tr "/" "\n" <<< "$charts_dir" | wc -l) + 1 ))
```

Reply:
Not sure if people have specific preferences... one could save the extra math operation by trading for an extra echo:
```suggestion
    local depth=$( (echo; tr "/" "\n" <<< "$charts_dir") | wc -l)
```

Reply:
I'm happy with either change; I thought direct math more clearly expressed intent but I appreciate that it's a little used bit of Bash syntax.

Reply:
I agree, direct math is better.